domains/_vercel.aadarsh.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Aadarsh0307",
+    "email": "aadarsh.rai0307@gmail.com"
+  },
+  "records": {
+    "TXT": [
+      "vc-domain-verify=aadarsh.is-a.dev,2e8150db7fb7f6bf781f"
+    ]
```

its just a single TXT record and for readability
```suggestion
    "TXT": "vc-domain-verify=aadarsh.is-a.dev,2e8150db7fb7f6bf781f"
```